packages/ember-metal/lib/meta.js

```diff
@@ -8,6 +8,16 @@ import EmptyObject from './empty_object';
 import { lookupDescriptor } from './utils';
 import symbol from './symbol';
 
+let metaCounters = window._metaCounters = {
```

Comment:
If we can get a heimdall release and use heimdall counters, I have a babel transform to strip it out for builds so that we can leave this sort of instrumentation in permanently.